wikidatarefisland/scraper.py

```diff
@@ -0,0 +1,84 @@
+import concurrent.futures
+from time import gmtime, strftime
+
+import extruct
+import requests
+from w3lib.html import get_base_url
+
+
+class Scaraper(object):
```

Comment:
I secretly wanted to do it later to avoid git conflicts but "Come what May"
![image](https://user-images.githubusercontent.com/5351225/80349779-092ed400-8870-11ea-9211-de5fca9eec89.png)

Inheriting from object makes it inheritable. Even it's not needed lots of times, it's recommanded to add it AFIAK.